### DISCUSSION TITLE
Mouse pointer not showing

### DISCUSSION THREAD

**Discussion by LucaMVieira** [prasanthrangan/hyprdots#1786] [Q&A] [answered]
I turn on my pc today to see that my mouse cursor was not showing in the screen. I could still used but i cannot see were it is.
I rebooted my pc a few times and tried removing the mouse usb and it didn't work.
In the sddm login screen I can see the mouse but when i login into hypr it stops showing.

The mouse does work on plasma wayland. It just doesn`t work on Hyde

Can anyone help me?

**Comment by kRHYME7** [comment, marked as answer]
```
cursor {
no_hardware_cursors = true
}
```

nvidia?

**Reply by LucaMVieira** [reply]
No, all amd

**Reply by kRHYME7** [reply]
`cursor:no_hardware_cursors = true`

already added this?

**Reply by LucaMVieira** [reply]
Yep just tried and worked perfectly

**Reply by TruWill99** [reply]
Just got this problem when I logged in today and the answer solved it.
Tyvm
